README.md

```diff
@@ -102,15 +102,13 @@ For more information about populating database, see the [Populating the Database
 5. To start the server in development mode, run the `npm start` command.
 6. To see the running Progressive Web Application (PWA), navigate to `http://localhost:8080/` .
 
-### Setting up a Production Environment
-
+To Build a Production Docker Image:<br>
 1. Clone or pull the `react-pwa-reference-storefront` repository to your directory.
 2. Run the `cd react-pwa-reference-storefront` command.
 3. Run the `docker build -t ep-store -f ./docker/prod/Dockerfile .` command.
 4. Push the `ep-store` image to the docker repository.
 
-### Setting up a Remote Production Environment
-
+To Run a Production Docker Image:<br>
```

Comment:
Did we want to merge this with the previous paragraph and have sub-titles? If this is the better option then I'm good with it too.